docs/user-guide/nodes.rst

```diff
@@ -113,44 +113,29 @@ To install *Nodes* child support, follow the instructions below.
 
  $ sudo yum install pulp-nodes-child pulp-nodes-consumer-extensions pulp-agent
 
-2. The communication between the child and parent nodes is secured using OAuth. The child node
-   must have OAuth enabled and configured. Please see :ref:`OAuth <oauth-config>` for instructions
-   on enabling and configuring OAuth.
+2. Configure ``/etc/pulp/server.conf`` on the child node to connect to the parent node via OAuth.
+   The communication between the child and parent nodes is secured using OAuth. Add the following to
+   the ``/etc/pulp/server.conf`` file on the child node.
 
 ::
 
  [oauth]
- enabled: true
- oauth_key: Xohkaethaama5eki
- oauth_secret: eePa7Bi3gohdir1pai2icohvaidai0io
-
-.. warning::
-
-   Do not use the key or secret given in the above example. It is important that you use unique and
-   secret values for these configuration items.
+ user_id:  <EDIT: admin user on parent node>
```

Comment:
This is the admin `user_id` on the child.  The `[oauth]` section in nodes.conf is for the *local* child REST API.  Not the parent.  Those settings are in `[parent_oauth]`.